e2e-tests/gef/cypress/integration/submit-to-ukef.spec.js

```diff
@@ -16,18 +16,17 @@ context('Submit to UKEF', () => {
         cy.apiFetchAllApplications(token);
       })
       .then(({ body }) => {
-        body.items.forEach((item) => {
-          applicationIds.push(item._id);
-        });
+        applicationId = body.items[0]._id;
```

Comment:
The previous test uses the third item in the array - is this a problem?

Reply:
I don't think so. The mock deals are almost exactly the same. The indexing in mock data loader is weird.... Probably don't need this change now.. Either way it doesn't seem to make a difference to the test now along with the other changes. I'd vote to not have this as a blocker and get this merged so we can progress other PRs

Reply:
Sounds good to me